components/src/core/components/PopOverButton/PopOverButton.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <div class="oxd-button-pop-over" v-click-outside="clickOutside">
+    <oxd-button
+      v-bind="$attrs"
+      @click.stop="onToggleDropdown"
+      @blur="onBlur"
+      @keyup.esc="onCloseDropdown"
+      @keydown.enter.prevent="onSelectEnter"
+      @keydown.down.exact.prevent="onSelectDown"
+      @keydown.up.exact.prevent="onSelectUp"
+    />
+    <div
+      class="oxd-pop-over-button-drop-down"
+      v-if="dropdownOpen && options.length > 0"
+      :class="dropdownClasses"
+      :style="dropDownStyle"
+      v-dropdown-direction
+      v-focus-first-element:return
```

Comment:
do we need `v-focus-first-element` here? since the index is 0 it always focus first item in dropdown?